coffeescript-loader/loader.js

```diff
@@ -1,6 +1,15 @@
+import { readFile } from 'fs/promises';
 import { readFileSync } from 'fs';
 import { createRequire } from 'module';
-import { URL, pathToFileURL } from 'url';
+import {
+  dirname,
+  resolve as resolvePath,
+} from 'path';
+import {
+  fileURLToPath,
+  pathToFileURL,
+  URL,
+} from 'url';
```

Comment:
This is ultimately going into the docs, so we should follow the style of the other examples. I think (?) they usually write these inline unless they need to wrap.

Also since we’re not importing some other `resolve`, we shouldn’t rename `path`‘s. And I think `URL` is a JavaScript global, not something we need to import.

```suggestion
import { dirname, resolve } from ‘path’;
import { fileURLToPath, pathToFileURL } from ‘url’;
```

Reply:
path's `resolve` must be renamed or namespaced to avoid conflicting with the resolve hook, which be renamed. I don't have much preference between renaming the import vs namespacing it (the rename seemed DRYer).

👍 for the rest :)

Reply:
> path’s `resolve` must be renamed or namespaced to avoid conflicting with the resolve hook, which be renamed. I don't have much preference between renaming the import vs namespacing it (the rename seemed DRYer).
> 
> 👍 for the rest :)

Ah, good point.